### PR TITLE
added http api for tidb

### PR DESCRIPTION
**Pull request by datahoecn**
added http api for tidb  @winkyao  PTAL

### CODE REVIEW

File: docs/tidb_http_api.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+#  TiDB HTTP API
+
+`TiDBIP` is the ip of the TiDB server. `10080` is the default status port, and it can be changed in tidb.toml when started.
```
Comment (winkyao) [inline docs/tidb_http_api.md]: when starts

File: docs/tidb_http_api.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+curl http://{TiDBIP}:10080/schema
+```
+
+> 7. Get schema info about db
+
+```
+curl http://{TiDBIP}:10080/schema/{db}
+```
+
+> 8. Get schema info about db.table
```
Comment (jackysp) [inline docs/tidb_http_api.md]: There is an api to get table schema by table id, e.g. curl http://{TiDBIP}:10080/schema?table_id={tableID}

File: docs/tidb_http_api.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+```
+curl http://{TiDBIP}:10080/schema/{db}/{table}
+```
+
+> 9. Get disk-usage info about db.table
+
+```
+curl http://{TiDBIP}:10080/tables/{db}/{table}/disk-usage
+```
+
```
Comment (jackysp) [inline docs/tidb_http_api.md]: It seems all mvcc information apis were missing.

File: docs/tidb_http_api.md [hunk 4]
```diff
@@ -56,3 +56,32 @@ curl http://{TiDBIP}:10080/schema/{db}/{table}
...
+> 13. Get MVCC Information by a hex value
+
+```
+curl http://{TiDBIP}:10080/mvcc/hex/{hexKey}
+```
+
+> 14. Get MVCC Information of index record key
+
+```
+curl http://{TiDBIP}:10080/mvcc/index/{db}/{table}/{index}/{handle}
```
Comment (jackysp) [inline docs/tidb_http_api.md]: Please add these `?${c1}={v1}&${c2}=${v2}` after handle, where c1 and c2 mean column name, and v1 and v2 represent column value.

Comment (jackysp) [reply docs/tidb_http_api.md]: Sorry for my inaccurate comment. I mean 14 should be 15. There is no api looks like http://{TiDBIP}:10080/mvcc/index/{db}/{table}/{index}/{handle}

**Comment by shenli** [comment]
LGTM

**Pull request merged**